### DISCUSSION TITLE
Добавить тире между исполнителем и названием трека в режиме экспортер

### DISCUSSION THREAD

**Discussion by F000x** [Chimildic/YaMuTools#61] [Идея]
На всех сервисах по импорту треков (включая tunemymusic.com и spotlistr) формат для импорта исполнитель - название трека. При клике по кнопке экспортер создается список исполнитель название трека (без тире), из-за чего непонятно, где заканчивается название исполнителя и начинается название трека.

**Comment by Chimildic** [comment]
Исходя из долгой практики, все лишние символы мешают поиску на конечной платформе

**Reply by kisoid** [reply]
Все лишние символы можно удалить в блокноте с помощью find & replace.
